projects/control/mobile_manipulation/mobile_manipulation_demo.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 1996-2020 OpenDR European Project
```

Comment:
This was wrong in a previous version of the wiki and master branch files but now all the instances are fixed.
```suggestion
# Copyright 2020-2021 OpenDR European Project
```